# HG changeset patch
# User Torsten <ttl@justmail.de>
# Date 1383252298 -3600
#      Thu Oct 31 21:44:58 2013 +0100
# Node ID 9c898878605397e6449f9c0be0f8ca7822952788
# Parent  e6317049225b7b13421a84c4584ff5528b1b2a25
*de_DE.ts: update for german gui translation

diff --git a/libgui/languages/de_DE.ts b/libgui/languages/de_DE.ts
--- a/libgui/languages/de_DE.ts
+++ b/libgui/languages/de_DE.ts
@@ -328,17 +328,17 @@ konnte nicht zum Schreiben geöffnet werden:
     <message>
         <location line="+53"/>
         <source>&amp;File</source>
         <translation>&amp;Datei</translation>
     </message>
     <message>
         <location line="+39"/>
         <source>&amp;Edit</source>
-        <translation>&amp;Barbeiten</translation>
+        <translation>&amp;Bearbeiten</translation>
     </message>
     <message>
         <location line="+22"/>
         <source>&amp;Preferences</source>
         <translation>&amp;Einstellungen</translation>
     </message>
     <message>
         <location line="+3"/>
@@ -978,17 +978,17 @@ zu: </translation>
     <message>
         <location line="+4"/>
         <source>Exit</source>
         <translation>Beenden</translation>
     </message>
     <message>
         <location line="+52"/>
         <source>&amp;Edit</source>
-        <translation>&amp;Editieren</translation>
+        <translation>&amp;Bearbeiten</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Undo</source>
         <translation>Rückgängig</translation>
     </message>
     <message>
         <location line="+7"/>
